qlib/contrib/strategy/signal_strategy.py

```diff
@@ -216,7 +217,7 @@ def filter_stock(li):
         buy = today[: len(sell) + self.topk - len(last)]
         for code in current_stock_list:
             if not self.trade_exchange.is_stock_tradable(
-                stock_id=code, start_time=trade_start_time, end_time=trade_end_time
+                stock_id=code, start_time=trade_start_time, end_time=trade_end_time, direction=OrderDir.SELL
```

Comment:
Thanks for your contribution.  @qianyun210603 
Not selling limit-up stocks and buying limit-down could be considered as a part of the strategy in A-stock.
Although current implementation seems a little weird,  but it is a reasonable results.

Your version is better. But if we merge it currently, all the previous results should be updated to make the code and results consistent (it will take a long time to update all the results).

Would it be better if we add it as a feature (a parameter in the init method) and keep the previous version as the default behavior? 
(Besides, TODOs to switch to your version should  be added in the docs/docstrings)

Reply:
Added a flag in `TopkDropoutStrategy` defaults to `True` to keep previous behaviour as suggested.